scalafix/v0.9.28/input/src/main/scala/fix/Case2.scala

```diff
@@ -0,0 +1,17 @@
+/*
+rule = v0_9_28
+ */
+package fix
+
+import scala.reflect.ensureAccessible
+import org.scalatest.ConfigMap
+import scalafix.testkit.SemanticRuleSuite
+
+trait BeforeAndAfterAllConfigMapAlt {
+  final def beforeAllAlt: Unit = ()
+}
+class RuleSuite extends SemanticRuleSuite with BeforeAndAfterAllConfigMapAlt {
+  val isSaveExpectField = ensureAccessible(classOf[SemanticRuleSuite].getDeclaredField("isSaveExpect"))
```

Comment:
I guess you encounter that in a real-life scenario? I wonder if we should consider exposing it in the API if the usage is legit

Reply:
Yes ! It was in scala-rewrite , not even sure they will upgrade. The other case scenario , is we fix most stuff, and we don't really care about really special things. In this case it's quiet easy to fix by hand for the author ?
What do you think about that ?

Reply:
> What do you think about that ?

About only supporting the most common use-case (what's on master) and not merge this PR? Well, since you did it, I don't see any reason not to merge this PR.

Reply:
I mean this pr doesnt support everything. The CI donest report the error but it's failing.

Reply:
https://github.com/scalacenter/scalafix.g8/pull/22/checks?check_run_id=2616970288

Reply:
I thought you had actually fixed the issue since I saw some updates in the rule itself. I had missed the first line of the description! Then yes, it's fine to provide a non-exhaustive/perfect script, it shouldn't be a blocker!

Reply:
Following up on the CI issue: https://github.com/scalacenter/scalafix.g8/pull/23

Reply:
could you rebase against https://github.com/scalacenter/scalafix.g8/pull/23 to confirm that we don't get false negatives?

Reply:
ok !

Reply:
I did it in https://github.com/scalacenter/scalafix.g8/pull/25 since you actually made the test pass